<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PositionPick</class>
 <widget class="QWidget" name="PositionPick">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Выбор позиций меню</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	background-color: #666699;
}</string>
  </property>
  <widget class="QWidget" name="ParentWidget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>600</width>
     <height>580</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
	background-color: #333333;
	border-radius: 7px;
}</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>10</y>
     <width>171</width>
     <height>491</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
	background-color: #333333;
	border-radius: 7px;
}</string>
   </property>
   <widget class="QSpinBox" name="quantity">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>40</y>
      <width>160</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Anonymous Pro</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QSpinBox{
	border: 1px solid black;
	background-color: #666699;
	color: white;
}</string>
    </property>
    <property name="maximum">
     <number>999</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Anonymous Pro</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	color: white;
	border: none;
}</string>
    </property>
    <property name="text">
     <string>Количество блюд</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>120</y>
      <width>160</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Anonymous Pro</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: white;
	background-color: #000033;
	border: 1px solid #333333;
	border-radius: 7px;
}

QPushButton:hover{
	background-color: #666699;
}

QPushButton:pressed{
	background-color:rgb(0, 0, 30);
	border: 3px inset
}</string>
    </property>
    <property name="text">
     <string>Сброс</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>510</y>
     <width>170</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Anonymous Pro</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color: white;
	background-color: #000033;
	border: 1px solid #333333;
	border-radius: 7px;
}

QPushButton:hover{
	background-color: #333333;
}

QPushButton:pressed{
	background-color:rgb(0, 0, 30);
	border: 3px inset
}</string>
   </property>
   <property name="text">
    <string>Закрыть меню</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
